### DISCUSSION TITLE
Need a improvement with Adetailer

### DISCUSSION THREAD

**Discussion by KiyeopYang** [continue-revolution/sd-webui-animatediff#528] [Ideas]
It seems that Animatediff and Adetailer don't work well together. When Adetailer does post-processing, it alters the images generated by Animatediff, resulting in only the parts modified by Adetailer appearing less smooth.

For example, only the face or hands might show flickering and wavy effects, while the background and other parts appear relatively natural in the animation.

Animatediff is expected to become widely used in the near future, and Adetailer has already become an essential tool. Therefore, it is crucial that these two tools integrate seamlessly.

**Comment by NergiZed** [comment]
Currently you just need to not use Adetailer, which is what I do.

However, I recall that there are workflows in comfyUI where Adetailer and AnimateDiff are integrated in a way that works and doesn't cause flickering, so it should be technically possible.
